### ISSUE TITLE
Frameworks without Info.plist are rejected

### ISSUE THREAD

**Issue by blender** [Carthage/Carthage#2500]
* carthage install method: [ ] .pkg, [] homebrew, [x] source
* `which carthage`: /usr/local/bin/carthage
* `carthage version`:  master (pre 0.30.0)
* `xcodebuild -version`:  Xcode 9.2
* Are you using `--no-build`? no
* Are you using `--no-use-binaries`? no
* Are you using `--use-submodules`? no
* Are you using `--cache-builds`? no
* Are you using `--new-resolver`? no

**Cartfile**
```
N/A
```

**Carthage Output**
```
Failed to write to /Users/Tommaso/Code/<redacted>/iOS/<redacted>/Carthage/Build/.Fabric.version: Error Domain=NSCocoaErrorDomain Code=4 "The folder “.Fabric.version” doesn’t exist." UserInfo={NSFilePath=/Users/Tommaso/Code/<redacted>/iOS/<redacted>/Carthage/Build/.Fabric.version, NSUserStringVariant=Folder, NSUnderlyingError=0x604000055d50 {Error Domain=NSPOSIXErrorDomain Code=2 "No such file or directory"}}
```

**Actual outcome**
Carthage did not write a `.version` file nor copy the contents of the archive to `Carthage/Build`

**Expected outcome**
Carthage should .write a `.version` file and copy the contents of the archive to `Carthage/Build`

**Comment by blender** [comment]
See discussion in https://github.com/Carthage/Carthage/issues/2491

**Comment by blender** [comment]
Fixed in 0.30.0
